tests/unittests/text-translator-quantization.test

```diff
@@ -0,0 +1,5 @@
+RUN: %text-translator -m %build_dir/en2gr  -keep-original-precision-for-nodes=Add -dump-profile=%t.yaml -quantization-schema=symmetric <<< "My name is Bob ." && \
+RUN: (%text-translator -m %build_dir/en2gr  -keep-original-precision-for-nodes=Add -load-profile=%t.yaml <<< "My name is Bob ." | FileCheck %s)
```

Comment:
In case someone wonders, `%t` tells LIT to generate a temporary and unique name for this test.
Thus, it is perfect to store intermediate files like quantization profile.